Uchu.World/Handlers/Commands/CharacterCommandHandler.cs

```diff
@@ -86,6 +87,7 @@ public string ChangeCoin(string[] arguments, Player player)
         {
             if (arguments.Length != 1) return "coin <delta>";
 
+            /// We parse this as an int instead of long, due to the max long causing bugs. Currency maxes out at a long.
```

Comment:
Really nitpicky but we generally only use double slash comments inline and use triple slash comments for doctrines